protos/deployment.proto

```diff
@@ -171,11 +171,19 @@ message DescribeDeploymentResponse {
 }
 
 message ListDeploymentsRequest {
+  enum SORTABLE_COLUMN {
+    unset = 0;
+    created_at = 1;
```

Comment:
I think you forgot to handle ‘unset’ again 
IIRC the first item in an enum is the default value in protobuf? how about just 
```
created_at = 0;
name = 1;
```

Reply:
same for the other list rpc below